docs/reference/setup/install/docker.asciidoc

```diff
@@ -125,6 +125,55 @@ curl --cacert http_ca.crt -u elastic https://localhost:9200
 ----
 // NOTCONSOLE
 
+==== Enroll additional nodes
+
+When you start {es} for the first time, the installation process configures a single-node cluster by default. This process also generates an enrollment token
+and prints it to your terminal. If you want a node to join an existing cluster,
+start the new node with the generated enrollment token.
+
+--
+.Generating enrollment tokens
+****
+The enrollment token is valid for 30 minutes. If you need to generate a 
+new enrollment token, run the
+<<create-enrollment-token,`elasticsearch-create-enrollment-token`>> tool on your
+existing node. This tool is available in the {es} `bin` directory of the Docker 
+container.
+
+For example, run the following command on the existing `es-node01` node to
+generate an enrollment token for new {es} nodes:
+
+[source,sh]
+----
+docker exec -it es-node01 /usr/share/elasticsearch/bin/elasticsearch-create-enrollment-token -s node
+----
+****
+--
+
+. In the terminal where you started your first node, copy the generated
+enrollment token for adding new {es} nodes. 
+
+. On your new node, start {es} and include the generated enrollment token.
++
+[source,sh,subs="attributes"]
+----
+docker run -e ENROLLMENT_TOKEN="<token>" --name es-node02 -p 9201:9200 --net elastic -it docker.elastic.co/elasticsearch/elasticsearch:{docker-image}
```

Comment:
```suggestion
docker run -e ENROLLMENT_TOKEN="<token>" --name es-node02 --net elastic -it docker.elastic.co/elasticsearch/elasticsearch:{docker-image}
```

I would avoid exposing the HTTP port of other nodes as it may be complex to understand and not always necessary.